### DISCUSSION TITLE
Need help in setting up stackql-server

### DISCUSSION THREAD

**Discussion by vwake7** [stackql/stackql#369] [Q&A] [answered]
Hi, 
      I am trying to setup stackql-server without success. I ran sudo docker run -d --rm -p 7432:7432 stackql/stackql-server, the container is up. But I cannot connect to the server in port 7432 using dbeaver postgres JDBC connection. I am getting  "serverVersion must not be null" error.  Can you please let me know what is the issue here or point to documentation, that will be great.

But I can exec in to the container and connect to 7432 using psq client. 

How can I connect using a sql client like dbeaver, any help is appreciated.

**Comment by jeffreyaven** [comment]
Hi @vwake7 can you connect via `psql`?

**Reply by vwake7** [reply]
yes I can connect using psql.

**Reply by jeffreyaven** [reply]
have you setup mTLS @vwake7

**Reply by vwake7** [reply]
Oh, I haven't. I will try out and let you know. Thanks @jeffreyaven

**Reply by jeffreyaven** [reply]
Hey @vwake7 you may need to update your version of stackql, the certs have expired, there is a new release `v0.5.571`, let us know how you go

**Reply by vwake7** [reply]
@jeffreyaven  I am using the docker image stackql/stackql-server, that still gives me the certificate error. Does this have the latest image.

**Reply by jeffreyaven** [reply]
What version of stackql are you on?

**Reply by vwake7** [reply]
I connected using psql to stackql/stackql-server:latest image, so from the stackql shell, What is the command to find the version?

**Reply by jeffreyaven** [reply, marked as answer]
try to pull the latest again now and run again @vwake7

**Comment by vwake7** [comment]
Sure I am caught up with something else, will try it and let you know.
Regards,
Vivek Anandaraman.
